frontend/app_flowy/packages/flowy_editor/lib/document/node.dart

```diff
@@ -184,8 +186,6 @@ class TextNode extends Node {
     return map;
   }
 
-  String toRawString() => _delta.operations
-      .whereType<TextInsert>()
-      .map((op) => op.content)
-      .toString();
+  String toRawString() =>
```

Comment:
The selection request will this method frequently. Actually the result can be cached.

Reply:
As a TODO. have added a comment.